Add New Cmdlet Docs (#11416)
MicrosoftDocs/office-docs-powershell: teams/teams-ps/teams/New-CsTeamsEventsPolicy.md

```
New-CsTeamsEventsPolicy [-Identity] <String> [-AllowWebinars <String>] [-AllowTownhalls <String>] [-AllowEmailEditing <String>] [-Description <String>]

> [-UseMicrosoftECDN -String] [-EventAccessType <String>] [-WhatIf] [-Confirm] [<CommonParameters>]

Reviewer: Should be <String>

Reviewer: string with < >

Accept wildcard characters: False
```

### -UseMicrosoftECDN

> This setting governs the admin disable this property and force the organizers to not create Town halls which use Microsoft eCDN even though they have been assigned a Teams Premium license.

Reviewer 2: ```suggestion
This setting governs whether the global admin disables this property and prevents the organizers from creating town halls that use Microsoft eCDN even though they have been assigned a Teams Premium license.

```
